lib/view/components/profile/request_container.dart

```diff
@@ -0,0 +1,99 @@
+import 'package:flutter/material.dart';
+
+class RequestContainer extends StatefulWidget {
+  final String requester;
+  final VoidCallback accept;
+  final VoidCallback reject;
+
+  const RequestContainer({super.key, 
+    required this.requester,
+    required this.accept, 
+    required this.reject,
+ });    
+
+  @override
+  State<RequestContainer> createState() => _RequestContainerState();
+  
+  }
+
+
+class _RequestContainerState extends State<RequestContainer> {
+  bool isDarkMode = true;
+
+  @override
+ Widget build(BuildContext context) {
+  isDarkMode=Theme.of(context).brightness == Brightness.dark;
+    return Container(
+  padding: const EdgeInsets.fromLTRB(12, 19, 12, 12), // Adjust spacing as needed
+  child: Center(
+    child: Column(
+      children: [
+        Text(
+          '${widget.requester} wants to connect',
+          textAlign: TextAlign.center,
```

Comment:
Perhaps its just me, but could we maybe move the text to the left hand side and have the button next to the text? let me know what you think

Reply:
causes an over flow for some screen sizes this was the only way i found to work for any screen size